src/pydata_sphinx_theme/assets/styles/variables/_fonts.scss

```diff
@@ -23,9 +23,12 @@ html {
   --pst-sidebar-header-font-size: 1.2rem;
   --pst-sidebar-header-font-weight: 600;
 
+  // Admonition styles
+  --pst-admonition-header-font-weight: 600;
+
   // Font weights
   --pst-font-weight-caption: 300;
-  --pst-font-weight-heading: 600;
+  --pst-font-weight-heading: 400;
```

Comment:
Could you be consistent in the naming structure? So either "heading-font-weight" or "font-weight-heading". Here they are mixed...

Reply:
Sure, I just chose to be consistent with `--pst-sidebar-header-font-weight` instead.

Reply:
Ah if this is a more general inconsistency in our docs, then don't worry about it

Reply:
It's an inconsistency in the code. Do you want me to revert the change?